mqtt.proxy.asyncapi/chart/Chart.yaml

```diff
@@ -3,4 +3,4 @@ name: zilla
 description: zilla mqtt.proxy example
 type: application
 version: 0.1.0
-appVersion: "latest"
+appVersion: "develop-SNAPSHOT"
```

Comment:
This is actually the mosquitto chart and the appVersion should correlate to the image tag for `eclipse-mosquitto` which in this case looks like it is pulling the `latest` version. This chart name and description should be updated since it is mosquitto and not zilla.